### DISCUSSION TITLE
Multiple files with the same updater?

### DISCUSSION THREAD

**Discussion by wcl-onespan** [absolute-version/commit-and-tag-version#41] [Q&A] [answered]
First off, great port, works better than the original...

Is there a way to specify multiple files to use the same custom updater, other than specifying each file individually in the versionrc.cjs?

This could be a pattern match like "*.html" or it could be that I need to use an array like ["index.html","login.html"] etc?  I have tried both of these options and neither seem to work, so I am not sure if it is possible.  If I can extend the filename field, that might be an option too since I am using a custom config, but I'm not sure where to start for that.

Any help or advice would be greatly appreciated.
-Will

**Comment by TimothyJones** [comment, marked as answer]
I don't believe this is possible at the moment. It seems like a natural use case, though. I'll create an issue for it.
